docs/excel/custom-functions-debugging.md

```diff
@@ -43,10 +42,10 @@ You can use the browser developer tools to debug custom functions in Excel Onlin
 ### Start debugging
 
 1. Open developer tools in the browser. For Chrome and most browsers F12 will open the developer tools.
-2. In developer tools, open your source code script file (functions.js or functions.ts). Your custom functions code may be located near the end of the file.
-3. In the custom function source code, apply a breakpoint by selecting a line of code.
+2. In developer tools, open your source code script file (functions.js or functions.ts).
+3. In the custom function source code, click in the margin to set a breakpoint on a line of code.
```

Comment:
```suggestion
3. In the custom function source code, select the margin next to the code line number to set a breakpoint on that line.
```

Reply:
"select" is confusing as it implies selecting text. "click in the margin" is what VS Code docs use for setting a breakpoint.